views/admin/network-admin-footer.php

```diff
@@ -6,21 +6,25 @@
 						<?php _e( 'Answer a short survey to let us know how we&#8217;re doing and what to add in the future.', 'jetpack' ); ?>
 					</div>
 					<div class="jp-survey-button-container">
-						<p class="submit"><?php printf( '<a id="jp-survey-button" class="button-primary" target="_blank" href="%1$s">%2$s</a>', 'http://jetpack.com/survey/?rel=' . JETPACK__VERSION, __( 'Take Survey', 'jetpack' ) ); ?></p>
+						<p class="submit">
+							<a id="jp-survey-button" class="button-primary" target="_blank" rel="noopener noreferrer" href="https://jetpack.com/survey/?rel=<?php echo esc_attr( JETPACK__VERSION ); ?>">
+								<?php _e( 'Take Survey', 'jetpack' ); ?>
+							</a>
+						</p>
 					</div>
 				</div>
 			</div>
 
 			<div id="jp-footer">
 				<p class="automattic"><?php _e( 'An <span>Automattic</span> Airline', 'jetpack' ) ?></p>
 				<p class="small">
-					<a href="http://jetpack.com/" target="_blank">Jetpack <?php echo esc_html( JETPACK__VERSION ); ?></a> |
-					<a href="http://automattic.com/privacy/" target="_blank"><?php _e( 'Privacy Policy', 'jetpack' ); ?></a> |
-					<a href="https://wordpress.com/tos/" target="_blank"><?php _e( 'Terms of Service', 'jetpack' ); ?></a> |
+					<a href="https://jetpack.com/" target="_blank" rel="noopener noreferrer">Jetpack <?php echo esc_html( JETPACK__VERSION ); ?></a> |
+					<a href="https://automattic.com/privacy/" target="_blank" rel="noopener noreferrer"><?php _e( 'Privacy Policy', 'jetpack' ); ?></a> |
```

Comment:
@dereksmart @oskosk looking at #8993  — should this multisite-dashboard's footer link still point to automattic.com/privacy or rather to that new `#/privacy` page?

Reply:
I think it should remain as-is, as the link text says "privacy policy" specifically.